manifests/params.pp

```diff
@@ -142,7 +142,12 @@
       $bindir               = pick($bindir, "/usr/lib/postgresql/${version}/bin")
       $datadir              = pick($datadir, "/var/lib/postgresql/${version}/main")
       $confdir              = pick($confdir, "/etc/postgresql/${version}/main")
-      $service_status       = pick($service_status, "/etc/init.d/${service_name} status | /bin/egrep -q 'Running clusters: .+|online'")
+      if versioncmp($::operatingsystemrelease, '8.0') >= 0 {
+        # Jessie uses systemd
+        $service_status = pick($service_status, "/usr/sbin/service ${service_name}@*-main status")
+      } else {
+        $service_status = pick($service_status, "/etc/init.d/${service_name} status | /bin/egrep -q 'Running clusters: .+|online'")
```

Comment:
how do we create the cluster on Debian?
it seems that ubuntu (14.04+) will need that too…